pkg/labels/consul_aggregator_test.go

```diff
@@ -181,7 +181,7 @@ func TestIgnoreIndividualWatch(t *testing.T) {
 	Assert(t).AreEqual(int64(2), aggreg.metWatchCount.Value(), "should currently have two watchers")
 
 	// iterate twice to show that we are not waiting on other now-closed channels
```

Comment:
hmm it seems like your comment appears both within the diff and here but mine only shows up in one or the other! anyway you're right i was looking at a different place with that identical comment